tests/back/Platform/Integration/CatalogVolumeMonitoring/Persistence/Query/CountUsersIntegration.php

```diff
@@ -38,20 +41,24 @@ protected function createUsers(int $numberOfUsers)
                 'catalog_default_locale' => 'en_US',
                 'user_default_locale' => 'en_US',
                 'roles' => ['ROLE_USER'],
-            ]);
+            ], $type);
             $i++;
         }
     }
 
 
     /**
      * @param array $data
+     * @param string $type
      * @return UserInterface
      */
-    protected function createUser(array $data = []) : UserInterface
+    protected function createUser(array $data = [], string $type) : UserInterface
     {
         $user = $this->get('pim_user.factory.user')->create();
         $this->get('pim_user.updater.user')->update($user, $data, []);
+
+        ($type === User::TYPE_APP) ? $user->defineAsUserApp() : null;
```

Comment:
I feel it more explicit like this.
```suggestion
        if ($type === User::TYPE_APP) {
            $user->defineAsUserApp();
        }
```